README.md

```diff
@@ -139,7 +139,7 @@ Publish `group/add-member` messages to a group of peers, which gives them all th
 - `feedIds` _[FeedId]_ - an Array of 1-15 different ids for peers (accepts ssb-uri or sigil feed ids)
 - `opts` _Object_ - with the options:
   - `text` _String_ - A piece of text attached to the addition. Visible to the whole group and the newly added people.
-- `cb` _Function_ - a callback of signature `(err, msg)`
+- `cb` _Function_ - a callback of signature `(err, Array<msg>)`
```

Comment:
you might wanna say what those messages are?

Reply:
maaybe. almost more keen on removing the return value, it should only be used in tests